isis/src/control/apps/jigsaw/jigsaw.xml

```diff
@@ -1513,5 +1513,106 @@
       </dataFiles>
     </example>
 
+    <example>
+      <brief>Run of jigsaw parameterized for Kaguya Terrain Camera (TC) images and <b>a relative control</b> network covering the Apollo 15 landing site. </brief>
+      <description>
+        <p>
+         A relative network is a network that has no points connected to a ground source, but connects overlapping images together. Since there is no connection to 
+         ground in the network, only camera specific parameters can be solved in the bundle. The required information to turn on spacecraft, radius, or other such
+         bundle solve parameters is not recommended.
+        </p> 
+        <p>
+         This relative bundle turns on and parameterizes the camera twist and camera acceleration solve parameters. Turning on camera twist allows for the bundle to solve 
+         for the cameras rotation around the bore sight axis and does not require parameterization. Setting the camera solve parameter to acceleration is useful for 
+         line scan imagers and requires uncertainties to be set for the angles (deg), angular velocity (deg/s), and angular accelerations (deg/s**2). These values were 
+         set with increasing constraint because of the increasing affect alterations of higher derivative parameters have on the bundle solution.
+        </p>
+        <p>
+         The ‘overexisting’ flag allows the bundle solution to approximate the camera solution with a polynomial function as opposed to storing every individual measurement. 
+         This saves time and memory required for the bundle solution at the expense of accuracy. If one required the accuracy, overexisting could be used until the bundle 
+         is converging in a satisfactory way and then turned off for the final solution.
+        </p>
+        <p>
+         This bundle also lowers the max iterations to 10 (from default 50). Lowering the max iterations does not affect the bundle solution. However, setting a lower 
+         iteration limit can serve as a flag if you expect your network to bundle quickly. Finally, the sigma0 convergence criteria was not changed from its default 
+         value, it was only explicitly stated in this call.
+        </p>
+      </description>
+      <terminalInterface>
+        <commandLine>
+        jigsaw fromlist=cubes.lis cnet=relative_reg_jig4_edit.net onet=relative_reg_jig5.net update=no file_prefix=jig5 sigma0=1.0e-10 maxits=10 \
```

Comment:
Add a not above that you have `update=no` because you are still evaluating the solution and do not want to apply it to your images just yet.